SubsoniciousKit/SubsoniciousKit/Authentication/AuthenticationManager.swift

```diff
@@ -28,14 +28,20 @@ public class AuthenticationManager: ObservableObject {
     private func bindResult() {
         $result
             .filter {
-                if case .success = $0 {
+                switch $0 {
+                case .success:
                     return true
+                default:
```

Comment:
case failure : there will never a new case in result :)

Reply:
because $0 is optional, there's also none to handle. I think it's easier to only keep default here. Only success deserves to return true :)

Reply:
ok, so IMO, I prefer to show .none in the switch :)

Reply:
As there is no code style convention about that in the project at the moment, I'm resolving the discussion :)